service/src/main/java/com/codeforcommunity/processor/SiteProcessorImpl.java

```diff
@@ -74,4 +94,45 @@ public void recordStewardship(
 
     record.store();
   }
+
+  @Override
+  public void deleteStewardship(JWTData userData, int activityId) {
+    StewardshipRecord activity =
+        db.selectFrom(STEWARDSHIP).where(STEWARDSHIP.ID.eq(activityId)).fetchOne();
+
+    if (activity == null) {
+      throw new ResourceDoesNotExistException(activityId, "Stewardship Activity");
+    }
+    if (!(activity.getUserId().equals(userData.getUserId())
+        || userData.getPrivilegeLevel().equals(PrivilegeLevel.SUPER_ADMIN)
+        || userData.getPrivilegeLevel().equals(PrivilegeLevel.ADMIN))) {
+      throw new AuthException(
+          "User needs to be an admin or the activity's author to delete the record.");
+    }
+
+    db.deleteFrom(STEWARDSHIP).where(STEWARDSHIP.ID.eq(activityId));
+  }
+
+  @Override
+  public StewardshipActivitiesResponse getStewardshipActivities(int siteId) {
+    List<StewardshipRecord> records =
+        db.selectFrom(STEWARDSHIP).where(STEWARDSHIP.SITE_ID.eq(siteId)).fetch();
+    List<StewardshipActivity> activities = new ArrayList<>();
+
+    records.forEach(
+        record -> {
+          StewardshipActivity stewardshipActivity =
+              new StewardshipActivity(
+                  record.getId(),
+                  record.getUserId(),
+                  record.getPerformedOn(),
+                  record.getWatered(),
+                  record.getMulched(),
+                  record.getCleaned(),
+                  record.getWeeded());
+          activities.add(stewardshipActivity);
+        });
```

Comment:
For this one, you can try `.fetchOneInto(StewardshipActivity.class)`. Depending on how the DTO was set up, you might also need to add a private empty constructor to the class, but we'll look at that if you're getting exceptions.

Reply:
Can you explain this a little more? fetchOneInto seems to fetch only one record, but I want a list of records.

Reply:
Oh whoops, I think the method i should have said would be something like 
`.fetchInto(SomeDto.class)`

Reply:
Is this worth it? Seems to just be confusing things when the current method works